src/translations/en.global.json

```diff
@@ -568,5 +568,8 @@
                 "remove": "If you lose access to your seed phrase, NEAR Inc. will be unable to assist you in recovery of your account and its funds."
             }
         }
-    }
+    },
+
+    "U2FNotSupported": "U2F browser support is needed for Ledger. Please use Chrome, Opera or Firefox with a U2F extension. Also make sure you're on an HTTPS connection.",
```

Comment:
@vgrichina I added support for two possible 'setup ledger' errors. I'm not sure how we'd add a 'fallback' error string for other possible errors though.

Reply:
@Patrick1904 here's how to handle missing translations in general, looks like can be overridden for individual `<Translate>` usage: https://ryandrewjohnson.github.io/react-localize-redux-docs/#handle-missing-translations

Another option is to check explicitly whether error `id` is in list of recognized error ids. Something like: 

```js
if (['U2FNotSupported', 'U2F_4'].indexOf(error.id) == -1) {
   // ... do whatever needed for missing translation
}
``` 

P.S. I think we shouldn't just use Ledger error codes in top level of translations. Makes sense to group them, e.g. under `errors.ledger` or smth like that.

Reply:
@vgrichina thank you I’ll fix that